Type definition for JSONAPISourceSettings broken in 0.15.11

After upgrading to 0.15.11, I'm getting the following type error

```
src/store.ts:88:3 - error TS2345: Argument of type '{ schema: any; name: string; }' is not assignable to parameter of type 'JSONAPISourceSettings | undefined'.
  Object literal may only specify known properties, and 'schema' does not exist in type 'JSONAPISourceSettings | undefined'.

88   schema: jsonApiSchema as any,
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

Here's how I'm instantiating `JSONAPISource`

```typescript
import { Schema } from '@orbit/data'
import JSONAPISource from '@orbit/jsonapi'

const jsonApiSchema = new Source({ ... })
const source = new JSONAPISource({
  schema: jsonApiSchema as any,
  name: 'source',
})
```

--- a/packages/@orbit/data/src/source.ts
+++ b/packages/@orbit/data/src/source.ts
@@ -3,6 +3,7 @@ import {
   evented, Evented, settleInSeries,
   Bucket,
   TaskQueue,
+  TaskQueueSettings,
   Task, Performer,
   Log
 } from '@orbit/core';

